yaksh/python_assertion_evaluator.py

```diff
@@ -70,23 +71,30 @@ def check_code(self):
             tb = None
             _tests = compile(self.test_case, '<string>', mode='exec')
             exec(_tests, self.exec_scope)
-        except AssertionError:
+        except TimeoutException:
+            raise
+        except Exception:
             type, value, tb = sys.exc_info()
             info = traceback.extract_tb(tb)
             fname, lineno, func, text = info[-1]
             text = str(self.test_case)
+
+            # Get truncated traceback
+            err_tb_lines = traceback.format_exc().splitlines()
+            stripped_tb_lines = []
+            for line in err_tb_lines:
+                if '.py' not in line:
```

Comment:
Not really, what if the exception had a .py inside it which was not a file.  Is there a test case that covers this specific condition?